build-system/app-index/template.html

```diff
@@ -51,6 +51,27 @@
     line-height: 40px;
     font-size: 18px;
     margin: 0;
+    display: flex;
+  }
+  .find-icon {
```

Comment:
I know it's not the point of this PR, but while my hands are tied with other stuff, would you be able to move the CSS into it's own single file, and include similar to how we are doing `<!--examples-->` ?

Reply:
I can then go ahead and do the same in my PR once I get around to updating it 😄

Reply:
Will do in a follow-up PR to prevent polluting the PR.